data.js

```diff
@@ -1,12 +1,10 @@
 var CarLot = (function(carlot) {
-  // Defining private varible to store parsed JSON data
-  var data = {}
   // Function to get JSON data and then call a function to build the HTML page
   carlot.loadInventory = (function() {
     var xhr = new XMLHttpRequest();
     xhr.addEventListener("load", function() {
-        data = JSON.parse(xhr.responseText).cars;
-        carlot.buildHTML(data)
+        let data = JSON.parse(xhr.responseText).cars;
+        carlot.populatePage(data)
```

Comment:
Calling this function still does not utilize the callback method. Instead trying accepting a callback in line 3. Then you would call `CarLot.loadInventory()` where you've defined your `populatePage()` and pass it into the call.